abnf/odata-aggregation-testcases.yaml

```
@@ -545,6 +557,20 @@ TestCases:
     Input: $apply=groupby((rolluprecursive($root/SalesOrganizations,SalesOrgHierarchy,ID)),aggregate($count as
       SalesOrgCount))
 
+  - Name: aggregate - groupby rollup recursive sub-hierarchy
+    Rule: queryOptions
+    Input: $apply=groupby((rolluprecursive(
+        $root/SalesOrganizations/$filter(Aggregation.isdescendant(
+          HierarchyNodes=$root/SalesOrganizations,
+          HierarchyQualifier='SalesOrgHierarchy',
+          Node=ID,
+          Ancestor='EMEA',
+          MaxDistance=2,
+          IncludeSelf=true)),
+        SalesOrgHierarchy,
+        SalesOrganization/ID)),
+      aggregate(Amount with sum as Total))
```

```suggestion
    Input: $apply=groupby((rolluprecursive(
        $root/SalesOrganizations/$filter(@sub),
        SalesOrgHierarchy,
        SalesOrganization/ID)),
      aggregate(Amount with sum as Total))
      &@sub=Aggregation.isdescendant(
          HierarchyNodes=$root/SalesOrganizations,
          HierarchyQualifier='SalesOrgHierarchy',
          Node=ID,
          Ancestor='EMEA',
          MaxDistance=2,
          IncludeSelf=true)
```
Is the slash in `$filter(...)` allowed here, or must we use an alias?

---

As this is in the query part of the URL, slashes are allowed as path separators.